v21.1/show-backup.md

```diff
@@ -34,6 +34,7 @@ Option       | Value | Description
 -------------+-------+-----------------------------------------------------
 `privileges` | N/A   |  List which users and roles had which privileges on each table in the backup. Displays original ownership of the backup.
 `encryption_passphrase`<a name="with-encryption-passphrase"></a> | [`STRING`](string.html) |  The passphrase used to [encrypt the files](take-and-restore-encrypted-backups.html) (`BACKUP` manifest and data files) that the `BACKUP` statement generates.
+`as_json`   |  N/A  | [Display the backup's manifest](#show-a-backups-manifest) as JSON in the response.
```

Comment:
I wonder if "manifest" is meaningful to the user or just us internally? should we say "...the backup's raw internal metadata" instead?